src/Core/Domain/CmsPageCategory/DTO/EditableCmsPageCategory.php

```diff
@@ -0,0 +1,186 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\CmsPageCategory\DTO;
+
+use PrestaShop\PrestaShop\Core\Domain\CmsPageCategory\Exception\CmsPageCategoryException;
+use PrestaShop\PrestaShop\Core\Domain\CmsPageCategory\ValueObject\CmsPageCategoryId;
+
+/**
+ * Class EditableCmsPageCategory
+ */
+class EditableCmsPageCategory
+{
+    /**
+     * @var array
+     */
+    private $localisedName;
+
+    /**
+     * @var bool
+     */
+    private $isDisplayed;
+
+    /**
+     * @var CmsPageCategoryId
+     */
+    private $parentId;
+
+    /**
+     * @var array
+     */
+    private $localisedDescription;
+
+    /**
+     * @var array
+     */
+    private $localisedMetaDescription;
+
+    /**
+     * @var array
+     */
+    private $localisedMetaKeywords;
+
+    /**
+     * @var array
+     */
+    private $localisedFriendlyUrl;
+    /**
+     * @var array
```

Comment:
The fact that it's an array was very strange to me 😅 I guess translation makes things complex

Reply:
We're gonna have to define a convention for this usecase. So far although this is an array, we keel a singular variable name. I dont know if it's relevant, but whatever we choose, we need to write it down.